test/automation/src/settings.ts

```diff
@@ -23,11 +21,12 @@ export class SettingsEditor {
 	}
 
 	async clearUserSettings(): Promise<void> {
-		const settingsPath = path.join(this.userDataPath, 'User', 'settings.json');
-		await new Promise<void>((c, e) => fs.writeFile(settingsPath, '{\n}', 'utf8', err => err ? e(err) : c()));
-
 		await this.openUserSettingsFile();
-		await this.editor.waitForEditorContents('settings.json', c => c === '{}');
+		await this.quickaccess.runCommand('editor.action.selectAll');
+		await this.code.dispatchKeybinding('Delete');
+		await this.editor.waitForTypeInEditor('settings.json', `{`); // will auto close }
+		await this.editors.saveOpenedFile();
+		await this.quickaccess.runCommand('workbench.action.closeActiveEditor');
```

Comment:
@sandy081 any issues with this change? This seems to make `clearUserSettings` more reliable and relies on UI instead of fs which was throwing for me with ENOENT which I didn't understand.